src/Codeception/Module/Doctrine2.php

```diff
@@ -236,29 +240,11 @@ public function flushToDatabase()
 
 
     /**
-     * Adds entity to repository and flushes. You can redefine it's properties with the second parameter.
-     *
-     * Example:
-     *
-     * ``` php
-     * <?php
-     * $I->persistEntity(new \Entity\User, array('name' => 'Miles'));
-     * $I->persistEntity($user, array('name' => 'Miles'));
-     * ```
-     *
-     * @param $obj
-     * @param array $values
+     * This is merely and alias to `haveInRepository()`.
      */
     public function persistEntity($obj, $values = [])
     {
-        if ($values) {
-            $rpa = new ReflectionPropertyAccessor();
-            $rpa->setProperties($obj, $values);
-            $this->populateEmbeddables($obj, $values);
-        }
-
-        $this->em->persist($obj);
-        $this->em->flush();
+        return $this->haveInRepository($obj, $values);
```

Comment:
Please use `Notification::deprecate` to emit deprecation message.
[Example](https://github.com/Codeception/Codeception/blob/3.0/src/Codeception/Util/Stub.php#L5-L12)

Reply:
Done.